### PR TITLE
Project detail page (add students)

### PR DESCRIPTION
**Pull request by Tiebe-Vercoutter**
Currently looks like this
afbeelding

afbeelding

Supports dnd: drag students to skills to add them

### CODE REVIEW

File: frontend/src/components/ProjectDetailComponents/ProjectCoaches/ProjectCoaches.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    return (
+        <CoachesContainer>
+            {editedProject.coaches.map((element, _index) => (
+                <CoachContainer key={_index}>
+                    <CoachText>{element.name}</CoachText>
+                    {editing && (
+                        <RemoveButton
+                            onClick={() => {
+                                const newCoaches = [...editedProject.coaches];
+                                console.log(_index);
```
Comment (stijndcl) [inline frontend/src/components/ProjectDetailComponents/ProjectCoaches/ProjectCoaches.tsx:26]: Leftover debug print
```suggestion
```

**Review by stijndcl — CHANGES_REQUESTED** [review]

**Comment by stijndcl** [comment]
Doesn't this branch from #395? Might want to change the base of the PR if so.

File: frontend/src/views/projectViews/ProjectDetailPage/ProjectDetailPage.tsx [hunk 2]
```diff
@@ -66,42 +70,176 @@ export default function ProjectDetailPage() {
...
+                        editing={editing}
+                    />
+
+                    <MoreInfoLink>
+                        <CreateButton
+                            showIcon={false}
+                            label="More info about this project"
+                            onClick={() => window.open(project.infoUrl)}
+                        />
+                    </MoreInfoLink>
```
Comment (stijndcl) [inline frontend/src/views/projectViews/ProjectDetailPage/ProjectDetailPage.tsx:214]: Seeing as the infoUrl is nullable, this should only be rendered if the url is not null. (Just use a basic `{project.infoUrl !== null && <Component />}` conditional render).
```suggestion
                    <MoreInfoLink>
                        <CreateButton
                            showIcon={false}
                            label="More info about this project"
                            onClick={() => window.open(project.infoUrl)}
                        />
                    </MoreInfoLink>
```

**Review by stijndcl — CHANGES_REQUESTED** [review]
InfoUrl is nullable. If this is used anywhere else in the code (I don't think it is), make sure to add a null check there as well.

**Comment by Tiebe-Vercoutter** [comment]
> Doesn't this branch from #395? Might want to change the base of the PR if so.

I think so yes. So first merge this one into #395 ?

**Comment by stijndcl** [comment]
> > Doesn't this branch from #395? Might want to change the base of the PR if so.
> 
> I think so yes. So first merge this one into #395 ?

Up to you, I just didn't want anyone accidentally merging this one first. It's your choice which one gets merged.

**Comment by Tiebe-Vercoutter** [comment]
> 
> Up to you, I just didn't want anyone accidentally merging this one first. It's your choice which one gets merged.

Yes okay this one can go into #395 and that one into develop. Solved the merge conflicts

**Pull request merged**